src/net/posix-stack.cc

```diff
@@ -942,7 +942,7 @@ std::vector<network_interface> posix_network_stack::network_interfaces() {
 
         auto pid = ::getpid();
 
-        sockaddr_nl local = { 0, };
+        sockaddr_nl local = {};
```

Comment:
Why not designate-initialize `local` instantly as well?

Reply:
ahh, indeed. changed.